src/index.js

```diff
@@ -19,8 +19,10 @@ const originalDocumentElement = document.documentElement
 function makeKCDWorkshopApp({imports, filesInfo, projectTitle, options = {}}) {
   const lazyComponents = {}
 
+  const componentExtensions = ['.js', '.md', '.mdx', '.tsx']
```

Comment:
I think we'll want `.ts` here as well

Reply:
How come? Can `.ts` can be used as an component?

Reply:
I mean these extensions are either run through loaders to make them renderable or contain JSX. I never seen JSX used within `.ts` (can it even be used inside `.ts`?)

Reply:
You can create a component without tsx is you don't use JSX in the file. I just want to be complete and not surprising for folks.

Reply:
With the `React.createElement` API right? Makes sense, changing.